File: Java/8. change_volume_level/app/src/main/java/com/example/change_volume_level/MainActivity.java
package com.example.change_volume_level;

import androidx.appcompat.app.AppCompatActivity;

import android.media.AudioManager;
import android.media.MediaPlayer;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.SeekBar;

public class MainActivity extends AppCompatActivity {
    MediaPlayer mediaPlayer;
    Button button;
    AudioManager audioManager;
    SeekBar seekBar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        button = findViewById(R.id.button);
        seekBar = findViewById(R.id.seekBar);

        mediaPlayer = MediaPlayer.create(this, R.raw.jerry_heil);
        audioManager = (AudioManager) getSystemService(AUDIO_SERVICE);
        int maxVolume = audioManager.getStreamMaxVolume(AudioManager.STREAM_MUSIC);
        seekBar.setMax(maxVolume);

        seekBar.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
            @Override
            public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
                audioManager.setStreamVolume(AudioManager.STREAM_MUSIC, progress, 0);
            }

            @Override
            public void onStartTrackingTouch(SeekBar seekBar) {

            }

            @Override
            public void onStopTrackingTouch(SeekBar seekBar) {

            }
        });
    }


    public void handlePlayTrack(View view) {
        if(mediaPlayer.isPlaying()) {
            mediaPlayer.pause();
            button.setText("paused");
        } else {
            mediaPlayer.start();
            button.setText("play");
        }
    }
}

File: LayoutWorkout/app/src/main/java/com/android/layoutworkout/MainActivity.kt
package com.android.layoutworkout

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.databinding.DataBindingUtil
import com.android.layoutworkout.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var ActivityMainBinding : ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

//        ActivityMainBinding = DataBindingUtil.setContentView(this, R.layout.activity_main)

//        ActivityMainBinding.button.setOnClickListener { changeText() }
    }

    private fun changeText() {
//        ActivityMainBinding.textView3.text = "Data binding"
    }
}